sr_unix_cm/gtcm_init.c

```diff
@@ -3,7 +3,7 @@
  * Copyright (c) 2001-2017 Fidelity National Information	*
  * Services, Inc. and/or its subsidiaries. All rights reserved.	*
  *								*
- * Copyright (c) 2017 YottaDB LLC. and/or its subsidiaries.	*
+ * Copyright (c) 2017-2018 YottaDB LLC. and/or its subsidiaries.*
```

Comment:
The commit comment has a lot about this module "**Define OMI_FREE() macro for better cleanup. Add to any place before error or return Initialize 'agname' and 'ag_pass'. Remove clauses that test memory allocation result. Note gtm_malloc() does not return if allocation fails**." But the change is just a copyright change. Are the changes missing somehow?

Update: Realized that this comment belongs to omi_prc_conn.c and not gtcm_init.c. Need an update in the commit message.

Reply:
Since no material changes happened to gtcm_init.c, the copyright change should also be removed.

Reply:
The newly pushed commit does not undo the gtcm_init() copyright changes.

Reply:
I'll try manually - last time I fixed it with a rebase on master which is something I don't want to do.